### ISSUE TITLE
[16.0] fetchmail_attach_from_folder: Missing ondelete on match_algorithm field

### ISSUE THREAD

**Issue by diggy128** [OCA/server-tools#3107]
## Module
fetchmail_attach_from_folder

## Describe the bug
match_algorithm is a required field and missing an ondelete policy

## To Reproduce
**Affected versions**: 16.0

1. Install fetchmail_attach_from_folder
2. Update module

**Expected behavior**
No error

**Additional context**
The exact error is: 
`'fetchmail.server.folder.match_algorithm': required selection fields must define an ondelete policy that implements the proper cleanup of the corresponding records upon module uninstallation.`

**Comment by NL66278** [comment]
There is no fetchmail_server_folder module in 16.0. Maybe you mean fetchmail_attach_from_folder? Anyway the match_algorithm field in taht module does NOT need an ondelete on the fetchmail.server.folder model, as this module defines the field for the first time, so there is no selection_add.

**Comment by diggy128** [comment]
@NL66278 
You are right in both points. edited the title and content regarding the module name and closed as invalid.
